### PR TITLE
Review dialog for pending connection request

### PR DESCRIPTION
**Pull request by crcommons**
- creates a new dialog for reviewing a pending connection request with visitor details, IDs, etc...
- replaces previous way to accepting/rejecting connection request in personal visitation requests
- adds it to the new professional requests

MISSING
- styling

### CODE REVIEW

**Review by hnqlv — APPROVED** [review]
Looks good for me! Just left two notes 👍🏼

**Comment by jackofsometrade** [comment]
@crcommons pushed styling and form validation changes, please review :pray:

**Review by jackofsometrade — APPROVED** [review]

File: src/components/Requests/ConnectionRequestsTable.tsx [hunk 1]
```diff
@@ -108,10 +111,13 @@ const ConnectionRequestsTable = ({
         title="ID Status"
         dataIndex="visitor"
         key="visitorIdentity"
-        render={(visitor: OrganizationMember) => (
-          // TO DO: needs styling and enum --> english mapping
-          <Tag>{visitor.identity.status}</Tag>
-        )}
+        render={(visitor: Visitor) =>
+          !visitor.identity ? (
```
[the remark this answers is not in the document]
Comment (cainlevy) [reply src/components/Requests/ConnectionRequestsTable.tsx:115]: seems like we should figure out whether ternary syntax is an important thing to align on as a group. i personally find them useful and readable in certain conditions such as this.

[the remark this answers is not in the document]
Comment (crcommons) [reply src/components/Requests/ConnectionRequestsTable.tsx:115]: Agreed. I decided to keep some and change others. We can chat about it in the engineering meeting!

**Pull request merged**